monkey/monkey_island/cc/ui/src/components/configuration-components/InfoField.js

```diff
@@ -12,9 +11,8 @@ class FieldWithInfo extends React.Component {
           <FontAwesomeIcon icon={faInfoCircle} style={{'marginRight': '5px'}}/>
           {this.props.schema.info}
         </div>
-        <ObjectField {...this.props} />
       </>);
```

Comment:
Do these empty tags do something, or are they left over cruft that should be removed? Does it change how anything is rendered if we remove them?

Reply:
From what I've read, "adjacent JSX elements must be wrapped in an enclosing tag." Since we got rid of `<ObjectField>`, I don't think the empty tag is necessary.